spec/generators/rspec/scaffold/scaffold_generator_spec.rb

```diff
@@ -40,7 +40,7 @@
       it { is_expected.to contain('renders a JSON response with errors for the post') }
     end
 
-    describe 'in an engine' do
+    xdescribe 'in an engine' do
```

Comment:
This suggests its a load order issue with the generators right?

Reply:
Yes. The call to this https://github.com/rspec/rspec-rails/blob/5d2c5eddd09a476baeca176ea2f596cbf290504c/spec/generators/rspec/scaffold/scaffold_generator_spec.rb#L45 breaks test in the same file that come after `describe 'in an engine'`.

Like if the stub was not restoring properly the behavior of `mountable_engine?`.

Reply:
Try running it in a sub process, at least that should isolate any poisoning of the globals.

Reply:
Good idea! It works!